### PR TITLE
connected archive to backend

### PR DESCRIPTION
**Pull request by LMWTT1230**

### CODE REVIEW

File: chessence-backend/models/user-services.js [hunk 1]
```diff
@@ -121,9 +121,25 @@ async function updateProfile(id, user, oldPwd) {
...
+async function getUsernameById(userId) {
+    //try {
+    const user = await userModel.findById(userId);
+
+    if (!user) {
+        return null;
+    }
+
+    return user.username;
+    // } catch (error) {
```
Comment (Xeygy) [inline chessence-backend/models/user-services.js:133]: why is this try catch block commented out?

**Review by Xeygy — APPROVED** [review]
Generally nice! One comment

**Review by ghervias — APPROVED** [review]
lgtm

**Review by e-acostag — APPROVED** [review]
LGTM

File: chessence-backend/models/user-services.js (hunk 1 above)
Comment (LMWTT1230) [reply chessence-backend/models/user-services.js:133]: Because it is gonna fail the coverage test.

**Review by knarfammussa — APPROVED** [review]
LGTM

**Pull request closed without merging**